docs/source/admin/quick_howto/oauth_login.rst

```diff
@@ -0,0 +1,90 @@
+		+------------------------------+------------+-------------------------------------------------------------------------------------------------------------------------------------------+
+		| oAuthCodeTokenUrl            | string     | URL to your OAuth provider's endpoint for exchanging the code (from oAuthUrl) for a token                                                 |
+		+------------------------------+------------+-------------------------------------------------------------------------------------------------------------------------------------------+
+		| clientSecret                 | string     | Client secret registered with OAuth provider to verify client, passed in with `client_secret` parameter                                   |
+		+------------------------------+------------+-------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+	.. code-block:: json
+		:caption: Example OAuth Configuration Properties In traffic_portal_properties.json
+
+		{
+			"oAuth": {
+				"_comment": "Opt-in OAuth properties for SSO login. See http://traffic-control-cdn.readthedocs.io/en/release-4.0.0/admin/quick_howto/oauth_login.html for more details. redirectUriParameterOverride defaults to redirect_uri if left blank.",
+				"enabled": true,
+				"oAuthUrl": "example.oauth.com",
+				"redirectUriParameterOverride": "",
+				"clientId": "",
+				"oAuthCodeTokenUrl": "example.oauth.com/oauth/token",
+				"clientSecret": ""
+			}
+		}
+
+- Update :file:`/opt/traffic_ops/app/conf/cdn.conf` property traffic_ops_golang.whitelisted_oauth_urls to contain all allowed domains for the JSON key set (Use ``*`` for wildcard):
+
+	.. table:: OAuth Configuration Property Definitions In cdn.conf
+
+		+--------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------+
+		| Name                     | Type               | Description                                                                                                     |
+		+==========================+====================+=================================================================================================================+
+		| whitelisted_oauth_urls   | Array of strings   | List of whitelisted URLs for the JSON public key set returned by OAuth provider.  Can contain ``*`` wildcards.  |
+		+--------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------+
+
+
+	.. code-block:: json
+		:caption: Example OAuth Configuration Properties In cdn.conf
+
+		{
+			"traffic_ops_golang": {
+				"whitelisted_oauth_urls": [
+					"example.oauth.com",
```

Comment:
Generally one of the reserved RFC example URLs are specified rather than a well-known one.

Reply:
done